### ISSUE TITLE
Core vocab is not available

### ISSUE THREAD

**Issue by berezovskyi** [oslc-op/website-content-negotiation#7]
A request to <http://open-services.net/ns/core#> with `Accept: text/turtle` returns 404 with the following error message:

> ENOENT: no such file or directory, stat &#39;/var/www/oslc.co/public/ns/core/core-vocab.ttl

**Comment by berezovskyi** [comment]
http://open-services.net/ns/cm# is available, for example.

I will investigate.

CC @jamsden @ndjc @axelreichwein

**Comment by berezovskyi** [comment]
Fixed in https://github.com/oslc-op/website-content-negotiation/commit/8e3cb9f43e41463072d92e56ea6dcf79f2574ce2. Apparently it was down since Sep 2019 when we deleted obsolete vocab files without realising the conneg did not point to the new vocabs but to the old: https://github.com/oslc-op/website/commit/3b5114ac06276a294dc9c941328d9c56c2c48603#diff-140937b0b1918cfa5802e4ff9645b6d8

I think this is a serious incident even though nobody raised a ticket (which reinforces my and Nick's opinion that no serious users rely on our live endpoint for their work/products). I suggest:

1. Go over the rest of the vocabs and test every Accept variant, make sure the response has a code of 200. I think a CURL script or similar is most appropriate for this.
2. Add an uptime check to ensure the endpoint content is always available. I am not suggesting something like PagerDuty with 24/7 on-call but a Slack notification if the endpoint is down should be manageable. I think we should be able to find some code I can readily put on my Raspberry Pi and/or my own VM.

@axelreichwein you have more ops experience, I am very open to your suggestions.
